.github/workflows/flake_finder.yml

```diff
@@ -8,6 +8,7 @@ on:
 jobs:
   e2e:
     name: E2E
+    if: github.repository_owner == 'submariner-io'
```

Comment:
Shouldn’t this also be added to the `upgrade-e2e` job? Or is there a way to disable a workflow globally?

Reply:
Ah missed it on this file, thanks.

Unfortunately I didn't find find a way to disable on the workflow level, but if you manage to find something it would be nice :)